src/lib/provable/crypto/poseidon.ts

```diff
@@ -33,6 +33,7 @@ type ProvableHashable<T> = Provable<T> & Hashable<T>;
 class Sponge {
   #sponge: unknown;
 
+  // TODO: implement constant version in TS. currently, you need to call `initializeBindings()` before successfully calling this
```

Comment:
I left this as TODO. `Poseidon.Sponge()` cannot be used before initializing bindings first, which could lead to users experiencing errors. I think the function is rarely used though, and there's an easy workaround: call `initializeBindings()`. The eventual solution is to implement `Sponge()` in TS (should be similar to `Poseidon.hash()`)

Reply:
Would it be easy/worthwhile to add an error message to call `initializeBindings()` until `Sponge()` is implemented in TS?